xs/src/libslic3r/Point.cpp

```diff
@@ -18,6 +18,12 @@ Point::operator==(const Point& rhs) const
     return this->coincides_with(rhs);
 }
 
+bool
+Point::operator<(const Point& rhs) const
+{
+    return this->x < rhs.x || this->y < rhs.y;
```

Comment:
This is needed by std::map. It's for ordering, not for comparing size. I'll move this to a more narrow scope.